### ISSUE TITLE
test_quantized_models.py times out within Nightly CI

### ISSUE THREAD

**Issue by seemethere** [pytorch/vision#1857]
Currently the nightly pipelines are failing when running `pytest .` on the `test/test_ops.py` tests.

Example CircleCI Logs:
https://app.circleci.com/jobs/github/pytorch/vision/85108

Log Excerpt:
```
+ pytest .
============================= test session starts ==============================
platform linux -- Python 3.6.10, pytest-5.3.5, py-1.8.1, pluggy-0.13.1
rootdir: $SRC_DIR
collected 286 items

test/test_backbone_utils.py ..                                           [  0%]
test/test_cpp_models.py sssssssssssssssssssssssssssssss                  [ 11%]
test/test_datasets.py ..........                                         [ 15%]
test/test_datasets_samplers.py .....                                     [ 16%]
test/test_datasets_transforms.py ..                                      [ 17%]
test/test_datasets_utils.py ..........                                   [ 20%]
test/test_datasets_video_utils.py ....                                   [ 22%]
test/test_functional_tensor.py ........                                  [ 25%]
test/test_hub.py sss                                                     [ 26%]
test/test_io.py ss............                                           [ 31%]
test/test_models.py .........s.........................................  [ 48%]
test/test_models_detection_utils.py .                                    [ 49%]
test/test_onnx.py sssssssssssss                                          [ 53%]
test/test_ops.py ..ss..ss..ss..ss..ss..ss..ss..ss.s...ss..ss             [ 68%]
Too long with no output (exceeded 10m0s): context deadline exceeded
```

This is what is currently effecting https://github.com/pytorch/pytorch/issues/33103

Also this may be related to https://github.com/pytorch/vision/issues/1528

cc @fmassa

**Comment by fmassa** [comment]
Thanks for the report!

I think this might be due to @zou3519 changes in https://github.com/pytorch/pytorch/pull/32495, which also caused nested tensors CI to fail, see https://github.com/pytorch/pytorch/issues/33091

**Comment by zou3519** [comment]
@fmassa does test_ops.py run c++ extensions?

**Comment by zou3519** [comment]
I don't think pytorch/pytorch#32495 is related because `test_ops.py` doesn't run c++ extensions directly. However, there's an easy way to test, assuming the CI runs on each PR (send a PR that disables building with ninja). I'll do that and see what happens

**Comment by seemethere** [comment]
Do we have any other leads on why this is happening? Perhaps adding some debugging information to the `pytest` run would provide some insight into what test it's actually timing out on

**Comment by seemethere** [comment]
After submitting https://github.com/pytorch/vision/pull/1880 I now believe the culprit we should be looking at is something that runs after `test/test_quantized_models.py::ModelTester::test_googlenet`.

From these [logs](https://app.circleci.com/jobs/github/pytorch/vision/87565):
```
test/test_ops.py::DeformConvTester::test_forward_cpu_contiguous PASSED   [ 67%]
test/test_ops.py::DeformConvTester::test_forward_cpu_non_contiguous PASSED [ 68%]
test/test_ops.py::DeformConvTester::test_forward_cuda_contiguous SKIPPED [ 68%]
test/test_ops.py::DeformConvTester::test_forward_cuda_non_contiguous SKIPPED [ 68%]
test/test_quantized_models.py::ModelTester::test_googlenet PASSED        [ 69%]

Too long with no output (exceeded 10m0s): context deadline exceeded
```

**Comment by zou3519** [comment]
Is this reproducible locally?

**Comment by seemethere** [comment]
I think it’s highly dependent on performance. If the test takes longer than 10 min to run then it’ll time out like this. Were there any tests within that file that were changed recently?

**Comment by fmassa** [comment]
`test_ops.py` does call C++ extensions, but as you mentioned it might not be `test_ops` the culprit.

If the issue is in `test_quantized_models`, I'd recommend checking with @raghuramank100 to see if anything has changed on the quantization side

**Comment by seemethere** [comment]
From my local testing the test that takes the longest is `Test Inception V3`  from the `test_quantized_models.py` file, which takes about `846s` on a devserver

Got this result from running on my branch that outputs junit logs
```bash
CU_VERSION=cpu PYTHON_VERSION=3.8 packaging/build_conda.sh
```

<img width="1404" alt="Screen Shot 2020-02-13 at 1 46 01 PM" src="https://user-images.githubusercontent.com/1700823/74481305-58678a00-4e67-11ea-9cbb-4750f9f1d4d4.png">


Results are here: [results.zip](https://github.com/pytorch/vision/files/4201309/results.zip)

This would lead me to believe that this test is the culprit.

**Comment by seemethere** [comment]
@fmassa @zou3519 Who would be the person to assign this issue to so we can re-add this test back to the nightly matrix?

**Comment by fmassa** [comment]
Assigning this to @raghuramank100 , who leads the quantization efforts

**Comment by datumbox** [comment]
This should be fixed by #3196.
